plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -653,21 +653,21 @@ private void RegisterExternalPreset(string name, string url, int width, int heig
     }
 
     private void RegisterPresets() {
-      RegisterPreset("Raidboss", width:1100, height:300, Strings.PresetRaidbossCombined, "raidboss");
-      RegisterPreset("Raidboss", width:1100, height:300, Strings.PresetRaidbossAlertOnly, "raidboss_alerts_only");
-      RegisterPreset("Raidboss", width:320, height:220, Strings.PresetRaidbossTimelineOnly, "raidboss_timeline_only");
-      RegisterPreset("Jobs", width:600, height:300, Strings.PresetJobs);
-      RegisterPreset("Eureka", width:400, height:400, Strings.PresetEureka);
-      RegisterPreset("Fisher", width:500, height:500, Strings.PresetFisher);
-      RegisterPreset("OopsyRaidsy", width:400, height:400, Strings.PresetOopsyRaidsy);
-      RegisterPreset("PullCounter", width:200, height:200, Strings.PresetPullCounter);
-      RegisterPreset("Radar", width:300, height:400, Strings.PresetRadar);
-      RegisterPreset("Test", width:300, height:300, Strings.PresetTest);
+      RegisterPreset("Raidboss", 1100, 300, "时间轴与触发器", "raidboss");
```

Comment:
I assume you didn't mean to add these plugin changes?

FWIW, if you could detect that the user had selected Chinese or was on the Chinese version, I'd be fine to upstream this into the plugin to do it dynamically.

Reply:
> I assume you didn't mean to add these plugin changes?
> 
> FWIW, if you could detect that the user had selected Chinese or was on the Chinese version, I'd be fine to upstream this into the plugin to do it dynamically.

Sorry, this PR was a mistake. I accidentally merged another branch into the branch of this PR yesterday. I will close this PR and reopen it later.